scenarios/aws/microVMs/microvms/libvirt_fs.go

```diff
@@ -220,11 +223,12 @@ func downloadRootfs(fs *LibvirtFilesystem, runner *Runner, depends []pulumi.Reso
 	}
 
 	if webDownload {
+		configPath := fmt.Sprintf("/tmp/aria2-%s.config", fs.fsNamer.ResourceName("aria2c"))
```

Comment:
```suggestion
		configPath := filepath.Join(os.TempDir(), fmt.Sprintf("aria2-%s.config", fs.fsNamer.ResourceName("aria2c")))
```

Reply:
`os.TempDir()` would return the default temporary directory on the system where pulumi is running.
On the otherhand, `configPath` targets the temp directory on the remote host where we are running the commands.
Given this, I think it may be better to keep `/tmp` as hardcoded.